src/main.py

```diff
@@ -606,8 +605,9 @@ def cleanup(signal, frame):   # pylint: disable=redefined-outer-name,unused-argu
 	pHandler.setup()
 	platform.setup()
 
+	logger.info("Checking Internet Connection ...")
 	while not internet_on():
```

Comment:
Why do we have that loop here? We're no retrying connection, `internet_on` is blocking and returns either `True` or `False`.